src/main/java/seedu/contax/logic/commands/RangeCommand.java

```diff
@@ -34,7 +33,9 @@
             + PREFIX_RANGE_TO + "TO *\n"
             + "Example: `" + COMMAND_WORD + " edit "
```

Comment:
```suggestion
            + "Example: `" + COMMAND_WORD + " editperson "
```
remember to change this too

Reply:
thx for the point